zerver/actions/message_edit.py

```diff
@@ -1,5 +1,5 @@
 import datetime
-from typing import TYPE_CHECKING, Any, Dict, Iterable, List, Optional, Set, TypedDict
+from typing import Any, Dict, Iterable, List, Optional, Set
```

Comment:
Removing `TYPE_CHECKING` here introduced a bug that was fixed in the next commit; I'm fixing that before merging the first commit, but be careful about this sort of thing, as it can often cause partially merging a PR to break CI.

Reply:
I'll keep that in mind. Thank you!!